ui/src/pages/home/app-card/app-card.tsx

```diff
@@ -246,69 +334,122 @@ export const AppCard = ({
+            <div className="chip-container">
+              <div className="menu-chip">
+                <Chip
+                  color={getStatusColor(appStatus)}
+                  label={appStatus}
+                  aria-label="open menu"
+                  id={id}
+                  children={undefined}
+                  size="small"
+                  className="chip-chip"
+                  sx={{
+                    ...statusStyles,
+                    '& .MuiChip-label': {
+                      color: statusStyles.color,
+                    },
+                  }}
+                />
+              </div>
+            </div>
+            <ContMenu id={`card-menu-${id}`} items={menuItems} />
+            {isStartOpen && (
+              <Dialog open={isStartOpen} onClose={setIsStartOpen}>
+                <DialogTitle>Start {title}</DialogTitle>
+                <DialogContent>{startModalBody}</DialogContent>
+              </Dialog>
+            )}
+            {isStopOpen && (
+              <Dialog open={isStopOpen} onClose={setIsStopOpen}>
+                <DialogTitle>Stop {title}</DialogTitle>
+                <DialogContent>{stopModalBody}</DialogContent>
+              </Dialog>
+            )}
+            {isDeleteOpen && (
+              <Dialog open={isDeleteOpen} onClose={setIsDeleteOpen}>
+                <DialogTitle>Delete {title}</DialogTitle>
+                <DialogContent>{deleteModalBody}</DialogContent>
+              </Dialog>
+            )}
+            <CardMedia>
+              {thumbnail ? (
+                <div className="img-overlay">
+                  <img src={thumbnail} alt="App thumb" />
+                </div>
+              ) : (
+                <div className="img-overlay">
+                  <img
+                    src="/assets/images/app-placeholder.jpeg"
+                    alt="App thumb"
+                  />
+                </div>
+              )}
+            </CardMedia>
+          </div>
+          <div className="card-cont-content">
+            <div className="chip-container">
+              <div className="menu-chip">
+                <Chip
+                  color="default"
+                  variant="outlined"
+                  label={framework}
+                  id={`chip-${id}`}
+                  size="small"
+                />
+              </div>
+              {isPublic && (
+                <div className="menu-chip">
+                  <Chip
+                    color="default"
+                    variant="outlined"
+                    label="Public"
+                    id={`chip-${id}-public`}
+                    size="small"
+                  />
+                </div>
+              )}
+            </div>
+            <div className="card-content-container">
+              <CardContent className="card-inner-content">
+                <span className="inline relative iconic">
+                  {' '}
+                  <LockIcon />{' '}
```

Comment:
Should this be conditionally displayed? I assume locked would be private, and some other icon would be for public?